server/model/status_page.js

```diff
@@ -46,7 +47,14 @@ class StatusPage extends BeanModel {
      */
     static async renderHTML(indexHTML, statusPage) {
         const $ = cheerio.load(indexHTML);
-        const description155 = statusPage.description?.substring(0, 155) ?? "";
+
+        let description155 = "";
+        if (statusPage.description) {
+            description155 = marked(statusPage.description)
+                .replace(/(<([^>]+)>)/gi, "")
```

Comment:
```suggestion
                .replace(/\<[^>]*\>/g, "")
```

That reflex for stripping the tags seems a bit complicated. What do you think about this one?

Reply:
@CommanderStorm 

Thanks for the review.

I agree with removing “ignore case” and removing “group brackets”.
However, I think “+” is preferable because a wildcard would also erase the <>, a non-html symbol pattern.

```js
// here
.replace(/\<[^>]+\>/g, "")
```

Reply:
LGTM

```suggestion
                .replace(/\<[^>]+\>/g, "")
```